assignment1q1/Assignment1q1.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package assignment1q1;

```

Comment:
You could consider using subpackages instead of top-level packages with two-part name, i.e. assignment1.q1 (or even better: assignment1.permutations) instead of assignment1q1.